apps/heft/src/utilities/CoreConfigFiles.ts

```diff
@@ -8,7 +8,6 @@ import { ICopyStaticAssetsConfigurationJson } from '../plugins/CopyStaticAssetsP
 import { IApiExtractorPluginConfiguration } from '../plugins/ApiExtractorPlugin/ApiExtractorPlugin';
```

Comment:
Heft's non-verbose logs should say something like:
```
Found a rig package with a profile in ./node_modules/heft-minimal-rig-test/profiles/default
```

Heft's verbose logs (or `--debug` maybe?) should print the full paths of the config files that are being loaded.

Reply:
I'd think that only verbose logs should log something. That information isn't useful for 99% of users.